### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by OHerasym**
### PR changes: ###

1. Add Windows build steps
2. Add QT build steps

Related: [APPLINK-25183](https://adc.luxoft.com/jira/browse/APPLINK-25183)

### CODE REVIEW

**Comment by OHerasym** [comment]
@dev-gh , @anosach-luxoft , Please review

File: README.md [hunk 1]
```diff
@@ -48,12 +48,70 @@ A quick guide to installing, configuring, and running an instance of the SDL Cor
...
+* Install [Visual Studio 2013](https://www.microsoft.com/en-us/download/confirmation.aspx?id=44914)
+
+##### Steps for Qt platform
+* Install [Visual Studio 2010](http://go.microsoft.com/?linkid=9709969)
+* Then install [Visual Studio SP1](https://www.microsoft.com/en-us/download/confirmation.aspx?id=23691)
+* Install [Qt framework](http://download.qt.io/official_releases/online_installers/qt-unified-windows-x86-online.exe).
+**Note:** It looks like starting from Qt5.6 there is no more msvc2010 32-bit support, so check whether such option is present. If no - use Qt5.5.
+ Create new environment variable called `SDL_QT_DIR` and assign your qt install dir to this variable. I.e `set SDL_QT_DIR=C:\Qt`
+
+##### Not required steps
```
Comment (dev-gh) [inline README.md]: @OHerasym 
I would propose to changed name to "Optional steps"

**Comment by dev-gh** [comment]
Reviewed

File: README.md (hunk 1 above)
Comment (OHerasym) [inline README.md]: @dev-gh , d9a0359, Done

**Comment by dev-gh** [comment]
Reviewed

**Comment by OHerasym** [comment]
@VVeremjova , @Kozoriz, @LuxoftAKutsan , @dtrunov , @pestOO , Please review

File: README.md [hunk 2]
```diff
@@ -67,12 +125,67 @@ Once SDL Core is compiled and installed you can start it from the executable in
...
+* Go to "Device Manager" from "Control Panel" or just right-click on Computer Icon and choose "Properties" and then click on "Device Manager".
+* You'll see an "Unknown Device" named something like "Bluetooth Peripheral Device".
+* Right-Click it and click on "Update Driver Software" or anyway you find it easy to bring up the Update driver window.
+* Click "Browse My Computer for Driver Software" button.
+* Click "Let Me Pick from a List of Device Drivers on my computer".
+* Select "Bluetooth Radio" from the list if it asked you to select and then in the next window, it will show a list of drivers with Company Names in one list and Drivers in another. In the Company List choose "Microsoft Corporation", not only Microsoft.
+* From the drivers list there may be one or more drivers with the name "Windows Mobile-Based Device Support" with different driver versions. Select one or the latest.
+* Ignore any warnings and keep pressing next and then Finish at last. If all goes well, the last screen will show the message that device driver is successfully installed.
+
+After all drivers has been installed allow SyncProxy service:
```
Comment (VVeremjova) [inline README.md]: @OHerasym `have` instead of `has`

File: README.md [hunk 3]
```diff
@@ -48,12 +48,70 @@ A quick guide to installing, configuring, and running an instance of the SDL Cor
...
+
+* Clone repository
+
+##### Steps for Windows platform
+
+* Go to directory with cloned repository
+* Run setup_VS2013_x64.cmd to generate solution for Visual Studio 2013.
+**Note:** In case Visual Studio has been installed to non-default location -- modify path in .cmd file.
+**Note:** Double click on file or run from windows command prompt. DO NOT run in "git bash".
+* After scrip execution finish, go to build directory which is located one directory upper and open smartDeviceLinkCore.sln file using Visual Studio 2013
```
Comment (dtrunov) [inline README.md]: @OHerasym typo script.

**Comment by dtrunov** [comment]
Reviewed

**Comment by VVeremjova** [comment]
@OHerasym Reviewed

**Comment by LuxoftAKutsan** [comment]
Reviewed

File: README.md (hunk 2 above)
Comment (OHerasym) [inline README.md]: @VVeremjova , db4e90b, Done

File: README.md (hunk 3 above)
Comment (OHerasym) [inline README.md]: @dtrunov , db4e90b, Done

**Comment by Kozoriz** [comment]
REviewed

**Comment by anosach-luxoft** [comment]
Reviewed

File: README.md [hunk 4]
```diff
@@ -48,12 +48,70 @@ A quick guide to installing, configuring, and running an instance of the SDL Cor
...
+
+##### Steps for Windows platform
+* Download [sqlite3 sources](https://www.sqlite.org/2015/sqlite-amalgamation-3090200.zip) and [sqlite3 binaries](https://www.sqlite.org/2015/sqlite-dll-win64-x64-3090200.zip). Extract all files to the same directory. Create new  environment variable `SDL_SQLITE_DIR` and assign path where sqlite sources and binaries were extracted to this variable.
+* Install [Windows SDK 7.1](https://www.microsoft.com/en-us/download/details.aspx?id=8442)
+* Install [Visual Studio 2013](https://www.microsoft.com/en-us/download/confirmation.aspx?id=44914)
+
+##### Steps for Qt platform
+* Install [Visual Studio 2010](http://go.microsoft.com/?linkid=9709969)
+* Then install [Visual Studio SP1](https://www.microsoft.com/en-us/download/confirmation.aspx?id=23691)
+* Install [Qt framework](http://download.qt.io/official_releases/online_installers/qt-unified-windows-x86-online.exe).
```
Comment (pestOO) [inline README.md]: @OHerasym, maybe better to provide the direct link from http://download.qt.io/archive/qt/5.5/5.5.1/ ?

**Comment by pestOO** [comment]
Reviewed

File: README.md (hunk 4 above)
Comment (OHerasym) [inline README.md]: @pestOO , b2a5a77, Done

**Pull request merged**